Generals/Code/GameEngine/Source/GameClient/MessageStream/CommandXlat.cpp

```diff
@@ -2734,6 +2734,15 @@ GameMessageDisposition CommandTranslator::translateGameMessage(const GameMessage
 
 		}	// end select previous worker
 
+		//-----------------------------------------------------------------------------------------
+		case GameMessage::MSG_META_SELECT_NEXT_IDLE_WORKER:
+			if (TheInGameUI)
```

Comment:
I think we do not need this null check. Hero selection below and Worker selection above also does not test for it.